app/channel.go

```diff
@@ -2525,7 +2525,7 @@ func (a *App) markChannelAsUnreadFromPostCRTUnsupported(postID string, userID st
 	if nErr != nil {
```

Comment:
Ooo .. we automatically get some added benefits here, as that's how the query flow seems to happen. Nice!

Just a small thing - I'd suggest to use the object returned on L2524, the one ignored by "_". Seems more natural as that's the latest updated one returned from the store.